export/index.test.js

```diff
@@ -235,3 +235,8 @@ describe("TKHQ", () => {
     ).rejects.toThrow('cannot create uint8array from invalid hex string');
   })
 });
+
+it("pads jwk keys to correct length", async () => {
+  expect(TKHQ.padBase64Url("wqVkltEnJAh2mJ5k9rAxEifyKv2BrAWscZ6QZDCS2Os", 32)).toBe("wqVkltEnJAh2mJ5k9rAxEifyKv2BrAWscZ6QZDCS2Os")
```

Comment:
We can add a little commentary to each of these test cases: the first does not change at all because its underlying byte representation occupies 32 bytes, and thus doesn't require padding

Whereas the second example requires a good amount of padding (you can essentially write out the mini process we went through to go from base64 --> hex / bytes --> base64)